pizza-order-showcase/pom.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.miragon.miranum.consulting</groupId>
+    <artifactId>miranum-pizza-order</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <name>miranum-pizza-order</name>
+    <packaging>pom</packaging>
+    <description>Miranum pizza order example</description>
```

Comment:
You renamed the project to pizza-order-showcase. Please adjust the groupId, artifactId, name and the description to match. Additionally we may also adjust the packages inside the services, because they still contain the "example" postfix.

Reply:
Oh my bad, the groupId is indeed correct